### ISSUE TITLE
Could not load file or assembly 'Twitchlab.Communication'

### ISSUE THREAD

**Issue by LeCanadienAlexo** [TwitchLib/TwitchLib#442]
Hi! 
Basically I've been making a small windows form app and using TwitchLib to connect to twitch and display the chat/send message to the chat.

Everything works perfectly, except it doesn't.

When I try to view my form, in visual studio, it tells me:

"Could not load file or assembly 'TwitchLib.Communication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The file can't be found.
and another error
The variable 'ucDashboard1' is either undeclared or wsa never assigned.

I don't understand what any of that means. I've been studying computer sciences for 3 years, but I still have a lot to learn, and this sounds out of my understanding.

My code is basically the Twitchlib.Client - CSharp example in the readme file of Twitchlib, nothing fancy or anything, but I use it in a UserControl that I placed on my form.

When I try to launch my app, it works perfectly, no error or anything, I can connect to twitch, send messages as my bot, view messages, everything. But I always have those two errors. The only way I can get rid of the errors is by replacing
```
public TwitchClient client;
        ConnectionCredentials credentials;

        public ucDashboard()
        {
            InitializeComponent();
            client = new TwitchClient();
            credentials = new ConnectionCredentials(botName, oAuth);
        }
```
by
```
        public TwitchClient client;
        ConnectionCredentials credentials;

        public ucDashboard()
        {
            InitializeComponent();
        }
```
basically removing the line where I set the client and credentials.

This makes no sense to me and I really can't wrap my head about what would be the issue here.

I saw quite a few people with errors regarding TwitchLib.Communication, but nothing that really looks relevant to my issue. This is my first time using the issue tab or Github, hope I'm doing it correctly.

**Comment by LeCanadienAlexo** [comment]
I have no idea why the formatting is acting weird, sorry for the strange codeblocks

**Comment by LeCanadienAlexo** [comment]
Issue fixed.

It was an internal problem with winForms and how it dealt with the initialization of the designer
